language/use-cases/document-qa/question_answering_documents_langchain_bigquery_vector_search.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<p>Is there a reason for this to use PaLM instead of Gemini?</p><p> </p><p>If you can switch this over to Gemini, then please move the notebook into the Gemini Directory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done. Hopefully I'm moving it to the right directory, and please review the naming.